deploy.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -ex
+pwd
+
+# Update fr.openfisca.org website to last revision
+git checkout master
+git pull
+
+# Rebuild website
+# User must have rights on '/node_modules' & '/.next'
+npm install
+npm run build
+
+# The current user must have been specifically allowed to run the next command
+# Use the visudo command to do so
+sudo systemctl restart fr.openfisca.org-beta.service
```

Comment:
Why beta ?

Reply:
Old name of the service. To be updated in an other PR along with `openfisca-ops`.

Reply:
This service is too recent to have an old name ^^. But ok for waiting for another PR.